docarray/dataclasses/types.py

```diff
@@ -21,6 +21,7 @@
 
 from docarray.dataclasses.getter import *
 from docarray.dataclasses.setter import *
+from docarray.dataclasses.enums import *
```

Comment:
why do we need to expose these ?

Reply:
Because AttributeType and DocumentMetadata are used in this file

Reply:
ah nvm thought all this is imported in docarray/dataclasses/__init__.py